### ISSUE TITLE
package.json configuration

### ISSUE THREAD

**Issue by mercmobily** [steelbrain/linter#1653]
Hi,

Sorry if this is not the right repository for this report.
I installed the linter in atom as a dependency of linter-js-standard. I added this to my package.json:

````
  "eslintConfig": {
    "parserOptions": {
      "ecmaVersion": 6
    },
    "rules": {
      "no-var": "error"
    }
  }
````

However, no matter what I do, I just can't get any of the rules I set here to apply.
I also tried to create a `.eslintrc` file in my project root -- no luck.

Am I missing something?

**Comment by Arcanemagus** [comment]
You'll need to file this over on `linter-js-standard`, `linter` has nothing to do with the configuration of individual providers.

Thanks!

**Comment by mercmobily** [comment]
Thanks ad sorry
